### ISSUE TITLE
[Release]: Accessibility statement dependency is missing when using released AARs

### ISSUE THREAD

**Issue by florentmaitre** [Orange-OpenSource/ods-android#885]

**Comment by B3nz01d** [comment]
@florentmaitre  do I need to test anything

**Comment by paulinea** [comment]
> @florentmaitre do I need to test anything

@B3nz01d No you can't.
